Lesson9/anna/Lesson9-aysel/styles.css

```diff
@@ -0,0 +1,214 @@
+* {
+  box-sizing: border-box;
+}
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.post {
+  display: grid;
+  margin: 100px auto;
+  grid-gap: 10px 10px;
+  grid-template-columns: 2fr 4fr 11fr 4fr 2fr;
+}
+img {
+  max-width: 100%;
+}
+
+.post > * {
+  grid-column: 3 / span 1;
+}
+.clock,
+.youtube-pic {
+  width: 100%;
+  grid-column: 2 / span 3;
+}
+
+h1 {
+  font-size: 42px;
+  font-weight: 700;
+  font-family: Arial, Helvetica, sans-serif;
+  line-height: 52px;
+  letter-spacing: -0.011em;
+}
+
+.first-pic {
+  border-radius: 50px;
+  position: relative;
+}
+.second-pic {
+  position: relative;
+  margin-left: -14px; /*i couldnt find any other way to put this pic on the first one*/
+  border-radius: 50px;
+  border: 2px solid rgb(255, 255, 255);
+}
+
+.publisher {
+  position: relative;
+  margin-left: 75px;
+  margin-top: -43px;
+  margin-bottom: 40px;
+  font-size: 14px;
+  font-weight: 400;
+  font-family: Arial, Helvetica, sans-serif;
+  line-height: 20px;
+}
+
+.publisher > a {
+  color: black;
+  text-decoration: none;
+}
+
+.publisher > a:hover {
+  text-decoration: underline;
+}
+
+.publisher > span {
+  color: gray;
+}
+
+.publisher > button {
+  color: rgb(12, 172, 100);
+  border: none;
+  background-color: transparent;
+}
+
+.bottom-information {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.left-bottom-information {
+  display: flex;
+  gap: 25px;
+}
+.right-bottom-information {
+  display: flex;
+  gap: 25px;
+}
+.bottom-information i {
+  color: rgb(130, 129, 129);
+  font-size: smaller;
+}
+hr {
+  border: 0.1px solid #c0bdbd;
+  margin: 0;
+}
+
+figcaption {
+  font-size: 14px;
+  text-align: center;
+  color: rgb(66, 65, 65);
+}
+
+.post > p,
+ol {
+  font-size: 20px;
+  font-family: "Times New Roman", Times, serif;
```

Comment:
Move the font family to the body element so it's applied to the entire page. For headings, change it to the different font.